pkg/config/testdata/override_gen.yaml

```diff
@@ -0,0 +1,33 @@
+kind: Configuration
+apiversion: ""
+metadata:
+    name: daprsystem
+    generatename: ""
+    namespace: default
+    selflink: ""
+    uid: ""
+    resourceversion: ""
+    generation: 0
+    creationtimestamp: "0001-01-01T00:00:00Z"
```

Comment:
I see some gen files cover timestamp, some not, do we need it?

Reply:
I don't remember how I created this file, but if it includes all this `metadata` is probably because I dumped it from the Go struct.

It should be irrelevant - the tests do not care about the `metadata` and just look at the `spec`.